Microsoft.Toolkit.Uwp.UI.Controls/Graph/ProfileCard/ProfileCard.Properties.cs

```diff
@@ -0,0 +1,198 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Graph
+{
+    /// <summary>
+    /// Defines the properties for the <see cref="ProfileCard"/> control.
+    /// </summary>
+    public partial class ProfileCard : Control
+    {
+        /// <summary>
+        /// Identifies the <see cref="GraphAccessToken"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty GraphAccessTokenProperty = DependencyProperty.Register(
+            nameof(GraphAccessToken),
+            typeof(string),
+            typeof(ProfileCard),
+            new PropertyMetadata(string.Empty, OnGraphAccessTokenChanged));
+
+        /// <summary>
+        /// Identifies the <see cref="UserId"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty UserIdProperty = DependencyProperty.Register(
+            nameof(UserId),
+            typeof(string),
+            typeof(ProfileCard),
+            new PropertyMetadata(string.Empty, OnUserIdPropertyChanged));
+
+        /// <summary>
+        /// Identifies the <see cref="DisplayMode"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty DisplayModeProperty = DependencyProperty.Register(
+            nameof(DisplayMode),
+            typeof(ViewType),
+            typeof(ProfileCard),
+            new PropertyMetadata(ViewType.PictureOnly, null));
+
+        /// <summary>
+        /// Identifies the <see cref="DefaultImage"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty DefaultImageProperty = DependencyProperty.Register(
+            nameof(DefaultImage),
+            typeof(BitmapImage),
+            typeof(ProfileCard),
+            new PropertyMetadata(null, OnDefaultImageChanged));
+
+        /// <summary>
+        /// Identifies the <see cref="DefaultTitleText"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty DefaultTitleTextProperty = DependencyProperty.Register(
```

Comment:
If the property is public, I would add it to the sample. Not mandatory, but it would make the control's sample more complete.

Reply:
@azchohfi Added to sample page